apps/designer/app/canvas/shared/styles.ts

```diff
@@ -250,6 +272,37 @@ const useUpdateStyle = () => {
       }
       utils.tree.setInstanceStyleMutable(rootInstance, id, updates, breakpoint);
     });
+
+    store.createTransaction([stylesContainer], (styles) => {
```

Comment:
We're calling `store.createTransaction()` twice in a row. I think this results in 2 HTTP requests. Can we call it once line this?

```
store.createTransaction([rootInstanceContainer, stylesContainer], 
(rootInstance, styles) => { ... })
```

/cc @kof as I only vaguely understand how `createTransaction` works.

Reply:
We put transactions in queue and send all at once.
I split because previous block will be completely removed in coming PRs.